ceph_deploy/hosts/rhel/mon/create.py

```diff
@@ -0,0 +1,24 @@
+from ceph_deploy.hosts import common
+from ceph_deploy.util import system
+from ceph_deploy.lib import remoto
+
+
+def create(distro, args, monitor_keyring):
+    hostname = distro.conn.remote_module.shortname()
+    common.mon_create(distro, args, monitor_keyring, hostname)
+    service = distro.conn.remote_module.which_service()
+
+    remoto.process.run(
+        distro.conn,
+        [
+            service,
+            'ceph',
+            '-c',
+            '/etc/ceph/{cluster}.conf'.format(cluster=args.cluster),
+            'start',
+            'mon.{hostname}'.format(hostname=hostname)
+        ],
+        timeout=7,
```

Comment:
A `RuntimeError` will be raised by remoto, which gets caught in cephdeploy.mon.mon_create().  That would cause the error to get logged, and an error to the user that it wasn't able to create the monitor.

mon_create() also checks to see that if the service *did* start successfully, that the actual mon status looks normal.